composer/utils/checkpoint.py

```diff
@@ -202,7 +202,7 @@ def _download_checkpoint(
             # every NODE needs the GLOBAL rank zero checkpoint
             path = _format_path_with_rank_zero(path)
             get_file(destination=rank_zero_checkpoint_filepath,
-                     path=path,
+                     path=_format_path_with_rank_zero(path),
```

Comment:
I assume this was bug? Seems like path should be formatted -- would like confirmation

Reply:
It is formatted above on line 203

Reply:
lol I'm blind, good catch